src/status_im/contexts/onboarding/create_password/view.cljs

```diff
@@ -205,12 +198,10 @@
         :passwords-match?          same-passwords?
         :empty-password?           empty-password?
         :show-password-validation? show-password-validation?
-        :on-input-focus            #(set-focused-input :password)
```

Comment:
This callback has been removed and its usages since it was executed `on-focus` and we had the `auto-focues` set to `true` for the input. It means it was a var that can be assumed as always set, it was never changed to other value